
<launch>
    <arg name="icp_config"/>
    <arg name="working_directory" default="$(env PWD)" />
    <arg name="lx" default="0.0" />
    <arg name="ly" default="0.0" />
    <arg name="lt" default="0.0" />

    <include file="$(find velodyne_pointcloud)/launch/32e_points.launch">
        <param name="frequency" value="10" />
    </include>

    <node name="cloud_matcher" pkg="pointmatcher_ros" type="matcher_service" >
      <param name="config" type="str" value="$(arg icp_config)" />
    </node>
    <node name="command_repeater" pkg="husky_trainer" type="command_repeater">
        <param name="input" value="/teach_repeat/desired_command" />
        <param name="output" value="/husky/cmd_vel" />
    </node>
    <node name="repeat_node" pkg="husky_trainer" type="repeat">
        <param name="working_directory" value="$(arg working_directory)" />
        <param name="lx" value="$(arg lx)" />
        <param name="ly" value="$(arg ly)" />
        <param name="lt" value="$(arg lt)" />
    </node>
</launch>
